model/task/task.go

```diff
@@ -650,6 +650,15 @@ func (t *Task) AddDependency(d Dependency) error {
 		}
 	}
 	t.DependsOn = append(t.DependsOn, d)
+
+	if d.TaskId == "" {
+		grip.Debug(message.Fields{
+			"ticket":     "EVG-16810",
```

Comment:
Unless we intend to keep this ticket open, it's actually possibly not helpful to include this here